composer.json

```diff
@@ -21,7 +21,7 @@
     "require": {
         "php": "^7.1",
         "container-interop/container-interop": "^1.0",
-        "monolog/monolog": "^1.0.2",
+        "monolog/monolog": "^1.3 || ^2.0",
```

Comment:
oh, very nice. i wasn't expecting this. please give me a couple of days to really dig in.

Reply:
I needed to define monolog:^1.3 because this version is first where psr-3 was implemented. And this interface is now a bridge between v1 and v2 in public interface of monolog.